include/ajax.tasks.php

```diff
@@ -336,20 +336,18 @@ function massProcess($action, $w=null)  {
                 $info[':title'] = sprintf('Reopen %s',
                          _N('selected task', 'selected tasks', $count));
```

Comment:
@Chefkeks I'm in favor of the longer phrases. I don't know what you mean with the mdash questions.